src/timepicker/timepicker.ts

```diff
@@ -26,15 +26,17 @@ const NGB_TIMEPICKER_VALUE_ACCESSOR = {
       left: 0.05em;
       position: relative;
       top: 0.15em;
-      -webkit-transform: rotate(-45deg);
       transform: rotate(-45deg);
+      -webkit-transform: rotate(-45deg);
+      -ms-transform: rotate(-45deg);
```

Comment:
Shouldn't the regular `transform` go after all of these vendor prefixes in order to override them?